cmake/dependencies/common.cmake

```diff
@@ -2,11 +2,22 @@ include(FetchContent)
 
 find_package(OpenGL QUIET)
 
+# When using the Visual Studio generator, it is necessary to suppress stderr output entirely so it does not interrupt the patch command.
+# Redirecting to nul is used here instead of the `--quiet` flag, as that flag was only recently introduced in git 2.25.0 (Jan 2022)
+if (CMAKE_GENERATOR MATCHES "Visual Studio")
+	set(git_hide_output 2> nul)
```

Comment:
[nit] Here and below, can you use 4 spaces instead of tabs to match the whitespacing of this file.

Reply:
Done.